src/uikit.cs

```diff
@@ -2140,6 +2140,7 @@ interface UIApplication {
 
 		[Deprecated (PlatformName.iOS, 13, 0, message: "Should not be used for applications that support multiple scenes because it returns a key window across all connected scenes.")]
 		[Deprecated (PlatformName.TvOS, 13, 0, message: "Should not be used for applications that support multiple scenes because it returns a key window across all connected scenes.")]
+		[NullAllowed]
```

Comment:
weird that xtro did not complain
and does not list that property

Reply:
I agreee, I did run xtro before the commit and no complains were given 🐟